packages/extension/src/newtab/MainFeedPage.tsx

```diff
@@ -32,34 +28,40 @@ export type MainFeedPageProps = {
 export default function MainFeedPage({
   onPageChanged,
 }: MainFeedPageProps): ReactElement {
-  const { windowLoaded } = useContext(ProgressiveEnhancementContext);
   const { user } = useContext(AuthContext);
   const { onboardingStep } = useContext(OnboardingContext) || {};
   const [feedName, setFeedName] = useState<string>('default');
+  const [activePage, setActivePage] = useState<string>(
+    feedName === 'default' ? '/popular' : feedName,
```

Comment:
Shouldn't this be: 

```suggestion
    feedName === 'default' ? '/' : feedName,
```

Reply:
And anyway, I think it's best to compute this value rather than have it as a state.
We can compute it using the different states we already have. `useMemo` for the win

Reply:
The path for the popular one is `/popular`, but the default also links to that.
So we need to make sure to set the activePage to `popular` if it's the default one.

I did change it to useMemo though :)